cloudinit/config/cc_ansible.py

```diff
@@ -226,4 +244,20 @@ def ansible_galaxy(cfg: dict, ansible: AnsiblePull):
     if not actions:
         LOG.warning("Invalid config: %s", cfg)
     for command in actions:
-        ansible.subp(command)
+        ansible.do_as(command)
+
+
+def ansible_controller(cfg: dict, ansible: AnsiblePull):
```

Comment:
Did something not get pushed? I don't see this used anywhere and I don't see the `ansible_controller` key consumed anywhere.

Reply:
Eek, that must have gotten dropped during rebase on upstream. I'll fix that.